core/src/main/java/jenkins/monitor/JavaVersionRecommendationAdminMonitor.java

```diff
@@ -44,6 +44,7 @@
 @Extension
 @Restricted(NoExternalUse.class)
 @Symbol("javaVersionRecommendation")
+@Deprecated(since = "TODO", forRemoval = true)
```

Comment:
This is going to be re-introduced soon, I don't think there's a point in removing it right now.